cumulus_library_suicidality_los/counts.py

```diff
@@ -6,6 +6,10 @@
 class SuicideLOSCountsBuilder(cumulus_library.CountsBuilder):
     display_text = "Creating suicide LOS counts..."
 
+    def __init__(self, study_prefix: str | None = None):
+        super().__init__()
+        self.study_prefix = "suicide_los"
```

Comment:
Why does this need to be manually specified?

Reply:
I think this is an issue with changes in how these are being loaded with the new approach. I'm going to investigate, but for these older studies, hard setting it should take care of this problem in the meantime.